values.yaml

```
@@ -23,6 +23,13 @@ httpsEgressSafelist:
 namespaces:
 - name: verify-main
 
+- name: verify-connector-node-metadata
```

```suggestion
- name: connector-node-metadata
```
This is the _verify_ cluster so we can call the new namespace just `connector-node-metadata` - just like we have `proxy-node-build`, etc

---

This is OK, ignore this comment